lib/cocoapods-core/specification/root_attribute_accessors.rb

```diff
@@ -146,6 +146,20 @@ def prepare_command
           command.strip_heredoc.chomp if command
         end
 
+        # @return [Bool] Indicates, that if use_frameworks! is specified, the
+        #         framework should include a static library.
+        #
+        def static_framework?
+          attributes_hash['static_framework']
+        end
+
+        # @return [Bool] Indicates, that if use_frameworks! is specified, the
+        #         framework should include a static library.
+        #
+        def static_framework
```

Comment:
My ruby is too weak to understand the reasons, but I get unit test failures like` ["[ERROR] [attributes] Unable to validate 'static_framework' (undefined method 'static_framework' for #<Pod::Specification name=nil>\nDid you mean?  static_framework?\n               static_framework=)."] ` if I remove it.